easybuild/easyconfigs/n/ncurses/ncurses-5.9-intel-2015a.eb

```diff
@@ -12,4 +12,7 @@ toolchainopts = {'optarch': True, 'pic': True}
 source_urls = [GNU_SOURCE]
 sources = [SOURCE_TAR_GZ]
 
+# enable UTF-8 version of ncurses
+configopts = '--enable-widec '
```

Comment:
I'd open a dedicated PR for this, rather than fiddling around in a PR that worked fine without this...

You can start with only touched a single ncurses in the PR, and expand as you go.